File: bronze/10807(개수세기)/10807.java
import java.util.Scanner;
public class Main{
    public static void main(String args[]){
        Scanner in = new scanner(System.in);
        int cnt=0;
        int n = in.nextInt();
        int[] arr = new int[101];
        for (int i=0;i<n;i++){
            arr[i] = in.nextInt();
        }
        int m = in.nextInt();
        for (int i=0;i<n;i++){
            if (arr[i]==m){
                 cnt++;
            }
        }
        System.out.println(cnt);
    }
}

File: bronze/10807(개수세기)/10807.c
#include <stdio.h>
int main(void){
    int n, m;
    int arr[101] = {};
    scanf("%d",&n);
    int cnt=0;
    for (int i=0;i<n;i++)
        scanf("%d",&arr[i]);
    scanf("%d",&m);
    for (int i=0;i<n;i++)
        if (arr[i]==m)
            cnt++;
    printf("%d\n",cnt);
    return 0;
}